packages/scene-composer/src/components/toolbars/common/styledComponents.ts

```diff
@@ -39,19 +39,20 @@ export const Icon = styled(PolarisIcon)<ToolbarItemIconProps>`
   transform: ${({ scale = 1, isMirrored }) => isMirrored && css`scaleX(${-scale});`};
 `;
 
-export const ToolbarItemContainer = styled.div<Pick<ToolbarItemOptions, 'isDisabled' | 'isSelected'>>`
+export const ToolbarItemContainer = styled.div<
+  Pick<ToolbarItemOptions, 'isDisabled' | 'isSelected'> & { height?: string }
+>`
   flex: none;
   position: relative;
   display: flex;
   align-items: center;
   justify-content: flex-start;
   min-width: 40px;
-  height: 40px;
+  height: ${({ height }) => height || '40px'};
```

Comment:
nit: Avoid the use of pixels as units.

Reply:
same here. need to make the parent component be responsive first